tests/dotnet/UnitTests/ProjectTest.cs

```diff
@@ -770,8 +770,8 @@ void ConfigureAssets (string project_path, string runtimeIdentifiers, bool start
 		void DeleteAssets (string project_path)
 		{
 			string xcassetsDir = Path.Combine (project_path, "../Assets.xcassets");
-			if (Directory.Exists (xcassetsDir))
-				Directory.Delete (xcassetsDir, true);
+			if (File.Exists (xcassetsDir))
+				File.Delete (xcassetsDir);
```

Comment:
@tj-devel709  as mentioned in our chat. File.Delete will not care if the file is not present. That check is redundant.